### DISCUSSION TITLE
GPU  Driver / CUDA problem

### DISCUSSION THREAD

**Discussion by BriceBeinsteiner** [nextpyp/.github#76] [Q&A] [answered]
Hi, 

I encounter a problem when using  MotionCor3/aretomo2 in nextPYP. 
During the preprocessing Job I get this error for aretomo2 : 

Info: skip device 0, system has unsupported display driver / cuda driver combination
I have 2 RTX3090 in this machine, and I tried with these driver/cuda combinations installed.

Nvidia 535.161.07 / CUDA12.2  and Nvidia 545.29.06 / CUDA 12.3

I assume you're embedding your own CUDA, but I can't find any driver version recommendations in your documentation.
nextPYP is installed on ubuntu and is not in a HPC

Thanks for your help, 

Brice

**Comment by abartesaghi** [comment]
Yes, nextPYP comes embedded with its own version of CUDA 11.4. Your versions of the Nvidia driver are fairly new and should be supported. Which version of nextPYP are you running?

**Reply by BriceBeinsteiner** [reply]
Thanks for your reply, I use nextPYP v0.6.2 for my test.

**Reply by abartesaghi** [reply, marked as answer]
We made some changes to the code that handles the GPU resource allocation in [0.6.4](https://nextpyp.app/files/pyp/0.6.4/docs/index.html) that may solve this issue. Let us know if you still have this problem after upgrading.
